Lecture_2/Ex4.c

```diff
@@ -0,0 +1,25 @@
+/******************************************************************************
+
+ Write a C program to compute the perimeter and 
+ area of a circle with a radius of 6 meters;
+
+*******************************************************************************/
+
+#include <stdio.h>
+#include <stdint.h>
+
+#define M_PI 3.14159265358979323846
```

Comment:
**M_PI** is defined in <math.h> include file. It is unnecessary to define it here